src/inmanta/data/__init__.py

```diff
@@ -5522,6 +5530,16 @@ async def list_notifications(
         ]
         return dtos
 
+    @classmethod
+    async def clean_up_notifications(cls) -> None:
+        environments = await Environment.get_list()
+        for env in environments:
+            time_to_retain_logs = await env.get(NOTIFICATION_RETENTION)
+            keep_notifications_until = datetime.datetime.now().astimezone() - datetime.timedelta(days=time_to_retain_logs)
+            LOGGER.debug(f"Cleaning up notifications in environment {env.name} that are older than {keep_notifications_until}")
```

Comment:
The job to cleanup old compile reports also logs with info level. To keep things consistent, we should do the same here.
```suggestion
            LOGGER.info(f"Cleaning up notifications in environment %s that are older than %s", env.name, keep_notifications_until)
```